README.md

```diff
@@ -41,6 +41,10 @@ Please consider the experience with this plug-in as experimental until Neovim 0.
 You can find the current roadmap [here](https://github.com/nvim-treesitter/nvim-treesitter/projects/1).
 The roadmap and all features of this plugin are open to change, and any suggestion will be highly appreciated!**
 
+*If you want to use this plugin with Neovim 0.5, please use the
+0.5-compat branch. Be aware though that most improvements will require
+neovim nightly.*
+
```

Comment:
Maybe also put the 0.5 install command next to the current one?

Reply:
I'd say we split them, just to make them separate and attract peoples eyes